prometheus_client/metrics_core.py

```diff
@@ -169,10 +201,17 @@ def __init__(self, name, documentation, count_value=None, sum_value=None, labels
         if labels is None:
             labels = []
         self._labelnames = tuple(labels)
```

Comment:
maybe `self._labelnames = tuple(labels) if labels else ()`

Reply:
Decent idea as a future cleanup, going to leave it out of this PR to keep the focus on types clear :).